docs/content/management.md

```diff
@@ -341,13 +341,13 @@ The JWT when decoded has a JSON payload of the following form:
 | `files[_].name` | `string` | Yes | Path of a file in the bundle. |
 | `files[_].hash` | `string` | Yes | Output of the hashing algorithm applied to the file. |
 | `files[_].algorithm` | `string` | Yes | Name of the hashing algorithm. |
-| `keyid` | `string` | No | Name of the key to use for JWT signature verification. |
+| `kid` | `string` | No | Name of the key to use for JWT signature verification (this is in the JWT header, not in payload). |
```

Comment:
Should we remove this line from the table? This is describing the payload, right?

Reply:
Removed.